tox.ini

```diff
@@ -154,3 +154,14 @@ commands =
 deps =
     pycrypto>=2.6
 passenv = {[testenv:system-tests]passenv}
+
+[testenv:flake8]
+commands = flake8 {posargs}
+deps = flake8-putty
+
+[flake8]
+exclude = .tox,.git,./*.egg,build,
+putty-ignore =
+  docs/conf.py : E402
```

Comment:
In this case, imports not being at the top.  There are some settings are defined prior to import (at least for django).

When I fix this pr, I will describe what's being ignored.